providers/core/core.oauth.js

```diff
@@ -98,8 +100,11 @@ OAuth.prototype.launchAuthFlow = function(authUrl, stateObj, continuation) {
     });
     return;
   }
-
-  this.ongoing[stateObj.state].launchAuthFlow(authUrl, stateObj, continuation);
+  if (interactive === undefined) {
+    interactive = true;
```

Comment:
OK - to me at least, defaulting to "true" feels like potentially surprising behavior, but I buy the convenience. I would suggest adding a "(default true)" annotation to the line 90 comment.